### PR TITLE
feat: Added query screen UI

### LINKED ISSUE #52: query section screen UI
image

**Describe the solution you'd like**
build the query section screen with below widgets:
1. query_message widget which can be customized with the following inputs to the constructor
    1. messageId
    1. message
    1. timestamp
    1. authorName (if the messge is sent by the logged in user then no need to show the name. also message sent by the loggedin_user is on the right and others will be on the left)
1. query_input which should support multiline
1. submit_query which is the button which sends the query to the backend.

### PR DESCRIPTION
**Pull request by sahil9001**
### Description

Added query screen UI.( My first flutter PR lol)
WhatsApp Image 2020-06-05 at 4 49 50 PM

**Type of Change:** (Delete irrelevant options)
- Code
- User Interface

**Code/Quality Assurance Only** (Delete irrelevant options)

- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
By `flutter doctor` and tested locally.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged

**Code/Quality Assurance**
- [ ] My code follows TDD clean architecture
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: lib/screens/query_section/widgets/query_message.dart [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+Widget queryMessage(
```
Comment (Hard-Coder05) [inline lib/screens/query_section/widgets/query_message.dart:4]: change the name of queryMessage to queryMessageCard that makes more sense

Comment (Hard-Coder05) [reply lib/screens/query_section/widgets/query_message.dart:4]: Over all nice code.. Everything looks cool!

Comment (sahil9001) [reply lib/screens/query_section/widgets/query_message.dart:4]: That's not a card, that's a message bubble, so will work like this. For reference, see #52

Comment (Hard-Coder05) [reply lib/screens/query_section/widgets/query_message.dart:4]: make it queryMessageBubble then queryMessage as a name of a widget makes no sense at all

**Review by Hard-Coder05 — APPROVED** [review]
Everything looking fine! Overall good code!

**Review by naveennvrgup — APPROVED** [review]
1. Great work @sahil9001 
1. Works as expected

**Pull request merged**